projects/templates/home.html

```diff
@@ -51,9 +53,13 @@
           <form class="d-flex" role="search">
             <input class="form-control me-2" type="search" placeholder="Search the blog" aria-label="Search">
               <i class="bi bi-search">
-                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="35" fill="currentColor" class="bi bi-search" viewBox="0 0 16 16">
-                  <path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/>
-                </svg>
+                <button class="btn btn-primary" style="color:#009a44">
```

Comment:
**Need to remove `style="color:#009a44"` and use Bootstrap class instead for green color**

Avoid using  inline styles where you can use  Bootstrap  classes
BS provides us with many solutions for coloring see here the documentation how to color BS buttons
https://getbootstrap.com/docs/5.0/components/buttons/